samples/deid.js

```diff
@@ -402,6 +402,61 @@ async function reidentifyWithFpe(
   // [END dlp_reidentify_fpe]
 }
 
+async function deidentifyWithReplacement(
+  callingProjectId,
+  string,
+  replacement
+) {
+  // [START dlp_deidentify_replacement]
+  // Imports the Google Cloud Data Loss Prevention library
+  const DLP = require('@google-cloud/dlp');
+
+  // Instantiates a client
+  const dlp = new DLP.DlpServiceClient();
+
+  // The project ID to run the API call under
+  // const callingProjectId = process.env.GCLOUD_PROJECT;
+
+  // The string to deidentify
+  // const string = 'My SSN is 372819127';
+
+  // The string to replace sensitive information with
+  // const replacement = "[REDACTED]"
+
+  // Construct deidentification request
+  const item = {value: string};
+  const request = {
+    parent: dlp.projectPath(callingProjectId),
+    deidentifyConfig: {
+      infoTypeTransformations: {
+        transformations: [
+          {
+            primitiveTransformation: {
+              replaceConfig: {
+                newValue: {
+                  stringValue: replacement,
+                },
+              },
+            },
+          },
+        ],
+      },
+    },
+    item: item,
+  };
+
+  try {
```

Comment:
We generally prefer to avoid the try/catch, and just let the code throw if it's going to throw.

Reply:
Thanks for reviewing! I was trying to keep the style consistent, all the other functions in this file use try/catch, so I went with that. Please lmk if you feel this is a strong anti-pattern, in which case maybe we should remove try/catch everywhere (perhaps in a separate PR).